operator/src/main/java/dev/responsive/k8s/controller/ControllerProtoFactories.java

```diff
@@ -48,6 +56,55 @@ public static ControllerOuterClass.EmptyRequest emptyRequest(final ResponsivePol
         .build();
   }
 
+  private static ProcessingRateDiagnoser processingRateDiagnoserFromK8sResource(
```

Comment:
these are all specific to `DemoPolicy` right? it might make sense to split this class into something that builds the proto just for the demo polcy and the general purpose stuff